utilities/src/main/java/io/syndesis/qe/bdd/validation/S3ValidationSteps.java

```diff
@@ -31,22 +33,9 @@ public S3ValidationSteps() {
         s3Utils = new S3Utils();
     }
 
-    @Then("^clean S3 to S3 scenario, remove two sample buckets with names: \"([^\"]*)\"")
-    public void cleanupS3(List<String> bucketNames) {
-        for (String bucket : bucketNames) {
-            String bucketName = S3BucketNameBuilder.getBucketName(bucket);
-            log.debug("deleting bucket with name: {}", bucketName);
-            s3Utils.deleteS3Bucket(bucketName);
-        }
-        TestSupport.getInstance().resetDB();
-    }
-
-    @Then("^clean S3, remove sample bucket with name: \"([^\"]*)\"")
-    public void cleanupS3bucket(String bucket) {
-        String bucketName = S3BucketNameBuilder.getBucketName(bucket);
-        log.debug("deleting bucket with name: {}", bucketName);
-        s3Utils.deleteS3Bucket(bucketName);
-        TestSupport.getInstance().resetDB();
+    @After("@cleanup-s3-after")
```

Comment:
this belongs into after hooks, for rest it is file: `RestTestHooks`. Lets keep it clear, it is not a validation step